.ignore

```diff
@@ -0,0 +1 @@
+/lib/
```

Comment:
Wouldn’t this affect the git sub modules though?

Reply:
Using `ag` and `rg` would not report results in the submodules, that's correct. I personally find myself always needing to add `--ignore lib` to my searches. If you think this is not the default people want though I'm happy to keep this change just local to my machine.

Reply:
You’re right, should be fine